CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.2.1
+
+This edge release sees the `linkerd-cni` plugin moved to
+`linkerd2-proxy-init` and released from that repository. An iptables
+improvement to `linkerd-cni` and `proxy-init` is the main focus. Other
+minor fixes are also included.
+
+* Upgrades `linkerd-cni` to v1.0.0 which is now published from
+  `linkerd2-proxy-init` and makes iptables rules idempotent (thanks @jim-minter!)
+* Upgraded `proxy-init` from v2.1.0 to v2.2.0 which makes `iptables` rules
+  idempotent (thanks @jim-minter!)
+* Add `server_port_subscribers` metrics to server and service watchers
+* Don't apply `waitBeforeExitSeconds` to control-plane pods
```

Comment:
It'd be more exact to say "control plane, viz and jager extension pods". Sorry the PR/commit title wasn't accurate...

Reply:
Thanks and I fixed the issues with mixed tense.